backend/functions/src/emailing/routes.ts

```diff
@@ -24,38 +24,34 @@ const router = express()
       FAIL -> res.data = 'Email send failure.' 
 
     */
-router.post('/send', (req, res) => {
+router.post('/send', async (req, res) => {
   const {
     emailAddress,
     authToken,
-    emailBody,
     emailSubject,
-    emailRcpts,
+    emailBody,
     courseId,
     group,
     template,
+    indivEmail,
   } = req.body
-  const message = createEmailAsJson(emailRcpts, emailSubject, emailBody)
 
-  sendMails(emailAddress, message, authToken, courseId, group, template)
-    .then((result) => {
-      if (result === 202) {
-        logger.info(
-          `Email sent successfully by ${emailAddress} to ${emailRcpts.toString()}`
-        )
-        res.json('Email send success.')
-      } else {
-        logger.error(
-          `** Likely auth error ** : Email failed to send by  ${emailAddress} to ${emailRcpts.toString()}`
-        )
-        res.json('Email send failure.')
-      }
-    })
-    .catch(() =>
-      logger.error(
-        `Email send request failed from ${emailAddress} to ${emailRcpts.toString()}`
-      )
+  try {
+    await sendStudEmails(
+      emailAddress,
+      authToken,
+      emailSubject,
+      emailBody,
+      courseId,
+      group,
+      template,
+      indivEmail
     )
+
+    res.status(200).json('Email send success')
+  } catch (err: any) {
+    res.status(400).json(err.message)
+  }
```

Comment:
Mainly because the logging needed access to emailRecpts, which is now inside of sendStudEmails. I guess I could change it so sendStudEmails returns an object that contains the status code/list of recipients so that the .then code can be moved back to routes.ts.

Reply:
I just mean that this try-catch code is equivalent to
```js
sendStudEmails(...)
  .then(() => res.status(200).json('Email send success') // also i do not like how these are different than the rest of the backend where it's an object with a success field, but i don't know if that's even preferable
  .catch((err) => res.status(400).json(err.message)
```
which looks more like other routes.

Small thing but to me it's preferable if sendStudEmails is not abbreviated, there's not much benefit to the shorter version other than 3 characters and we're not trying to save those 3 letters. Any abbreviation has a small risk of being confusing in the future.